src/js/views/patients/patient/flow/flow_views.js

```diff
@@ -161,6 +161,10 @@ const ActionItemView = View.extend({
     Radio.trigger('event-router', 'flow:action', this.model.get('_flow'), this.model.id);
   },
   onEditing(isEditing) {
+    const isSelected = this.state.isSelected(this.model);
+
+    if (isSelected) return;
+
```

Comment:
`onEditing()` adds or removes the `.is-selected` class when the edit action sidebar opens or closes.

I added an `if (isSelected)` check here that `return`s if the action is currently selected via its checkbox.